core/src/main/java/org/elasticsearch/action/update/UpdateRequest.java

```diff
@@ -112,17 +112,17 @@ public ActionRequestValidationException validate() {
             validationException = addValidationError("id is missing", validationException);
         }
 
-        if (!(versionType == VersionType.INTERNAL || versionType == VersionType.FORCE)) {
-            validationException = addValidationError("version type [" + versionType + "] is not supported by the update API", validationException);
-        } else {
+        if (versionType != VersionType.INTERNAL) {
```

Comment:
I think the if /else structure was to prevent complaining about irrelevant stuff?